ocaml/idl/datamodel.ml

```diff
@@ -6600,11 +6600,13 @@ let vdi_open_database = call
 let vdi_checksum = call
     ~name:"checksum"
     ~in_oss_since:None
-    ~in_product_since:rel_boston
+    ~lifecycle:
+      [Published, rel_boston, "For internal use only."
+      ;Changed, rel_inverness, "Available in the published documented API; otherwise unchanged."
```

Comment:
I really like this list formatting, but the convention is that we also have spaces before the list items:
```ocaml
~lifecycle:
  [ Published, rel_boston, "For internal use only."
  ; Changed, rel_inverness, "Available in the published documented API; otherwise unchanged."
  ]
```